app/workers/address_average_deposit_time_generator.rb

```diff
@@ -9,30 +13,22 @@ def perform
     end
   end
 
-  private
+  # return average deposit time of specific address (unit in day)
+  def cal_average_deposit_time(address, ended_at = CkbUtils.time_in_milliseconds(Time.current))
```

Comment:
This update fixes the logic to accumulate deposit time of a deposited cell.

In previous codes, the deposit time is accumulated to the address owning the withdrawing cells, missing an edge case that depositor transfer the cell to another address.

Say, `addr_a` deposits to `addr_b` and withdrawn by `addr_c`, now `addr_c` owns the cell and gets the accumulated deposit time. However, `addr_b` is the one keeping a locked cell, the time should be claimed by `addr_b`.

This PR fixes this hypothesis and calculate deposit time based on the depositor's address